src/dev/precommit_hook/casing_check_config.js

```diff
@@ -58,6 +57,11 @@ export const IGNORE_FILE_GLOBS = [
 
   // filename required by api-extractor
   'api-documenter.json',
+
+  // TODO fix file names in APM to remove these
+  'x-pack/plugins/apm/public/**/*',
+  'x-pack/plugins/apm/scripts/**/*',
+  'x-pack/plugins/apm/e2e/**/*',
```

Comment:
Is there any automated way (`eslint --fix`) to update the filenames?

Reply:
not for `scripts` or `e2e`, i started working on a code mod to update all the imports with the new paths, but then it was suggested by the platform team to ignore them for now and address it in another PR